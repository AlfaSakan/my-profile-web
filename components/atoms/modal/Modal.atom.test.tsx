import renderer from 'react-test-renderer';
import Modal from './Modal.atom';

describe('modal component test', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(
        <Modal isOpen>
          <p>mlampclmspc</p>
        </Modal>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
