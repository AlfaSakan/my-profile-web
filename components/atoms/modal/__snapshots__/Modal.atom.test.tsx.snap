// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modal component test should render correctly 1`] = `
<div
  className="fixed h-full w-full bg-black-200 top-0 flex items-center justify-center"
>
  <p>
    mlampclmspc
  </p>
</div>
`;
